qutip/solver/__init__.py

```diff
@@ -16,3 +16,5 @@
 from .brmesolve import *
 from .krylovsolve import *
 from .parallel import *
+import qutip.solver.sode
```

Comment:
Is the idea that one can then access `qutip.solver.sode.XXX` does this do this? It looks like it will make `XXX` available as `qutip.solver.sode.qutip.solver.XXX`? Maybe I am missing something.

Reply:
Updated to `import qutip.solver.sode as sode`, I thought this renaming was automatic...